README.md

```diff
@@ -190,3 +190,18 @@ If you feel our code or models helps in your research, kindly cite our papers:
   year={2020}
 }
 ```
+
+# New backbone shufflenet module.
```

Comment:
Please remove this training bash from README, we will add it later to docs page (https://cv.gluon.ai/model_zoo/classification.html). Thank you.